test/functional/test_framework/test_framework.py

```diff
@@ -155,6 +159,15 @@ def main(self):
             exit_code = self.shutdown()
             sys.exit(exit_code)
 
+    def run_custom_test(self):
+        for method_name in self.options.test_methods:
```

Comment:
Since the argument is a list named `test_methods`, let's call this function `run_custom_tests`?

Reply:
Done, thanks for reviewing